src/sos/workflow_engines.py

```diff
@@ -182,17 +178,19 @@ def purge_workflows(self,
                         tags=None,
                         verbosity=2):
         try:
-            return self.agent.check_output(
-                "{} purge {} {} {} {} {} -v {}".format(
-                    self.agent.config.get("sos", "sos"),
-                    " ".join(workflows),
-                    "--all" if purge_all else "",
-                    f"--age {age}" if age is not None else "",
-                    f'--status {" ".join(status)}'
-                    if status is not None else "",
-                    f'--tags {" ".join(tags)}' if tags is not None else "",
-                    verbosity,
-                ))
+            check_output2 = (
+                f"{self.agent.config.get('sos', 'sos')} purge"
+                f"{' '.join(workflows)}"
+                f"{'--all' if purge_all else ''}"
+                f"--age {age}" if age is not None else ''
+                f"--status {' '.join(status)}"
+                if status is not None else ''
```

Comment:
This will not work since the stuff before "if" and after "else" will become two big string.